neo4j-jdbc/src/main/java/org/neo4j/jdbc/impl/ListNeo4jResultSet.java

```diff
@@ -103,15 +103,17 @@ private Object get(String columnLabel) throws SQLException {
 	}
 
 	@Override public boolean getBoolean(int columnIndex) throws SQLException {
-		return false;
+		Object value = this.get(columnIndex);
+		return (boolean) value;
```

Comment:
what's the reason for changing these now?

Reply:
Good question I remember that we changed it in 5.0 so I brought the same code base here. I guess it's because we use it right now. But not sure